packages/backend/src/assets/ai.json

```
@@ -69,6 +69,23 @@
       ],
       "backend": "llama-cpp"
     },
+    {
+      "id": "chatbot-java",
```

```suggestion
      "id": "chatbot-java-quarkus",
```

so it'll match suggested `exampleName-language-framework/platform`

---

@rhatdan / @jeffmaury agree with renaming this recipe (I'd first change it in https://github.com/containers/ai-lab-recipes for consistency)?

---

yes please change first into ai-lab-recipes for consistency